tools/buildsys/src/main.rs

```diff
@@ -257,6 +258,21 @@ fn build_variant(args: BuildVariantArgs) -> Result<()> {
     Ok(())
 }
 
+fn repack_variant(args: RepackVariantArgs) -> Result<()> {
+    let manifest_file = "Cargo.toml";
+
+    let manifest = ManifestInfo::new(args.common.cargo_manifest_dir.join(manifest_file))
+        .context(error::ManifestParseSnafu)?;
+
+    supported_arch(&manifest, args.common.arch)?;
```

Comment:
For consistency, this should also include the `if manifest.included_packages().is_some()` check from `build_variant` (because otherwise how could an image have been built?)

Although that check is kind of annoying so you could instead remove it in a preliminary refactoring commit.